README.md

```diff
@@ -20,62 +20,88 @@ Minimal and straightforward when doing queries
 ```python
 from nmi_mysql import nmi_mysql
 ```
-- Initialization: Accepts two parameters, first being the config object and the second specifying if autoconnect to db is enabled. If set to false, call `con.connect()` 
+
+- Initialization: Accepts one parameter, which is the config object
 
 ```python
-try:
-    con = nmi_mysql.DB(conf, True)
-except Exception as err:
-    print(err)
+db = nmi_mysql.DB(conf)
+```
+
+- Connection
+
+```python
+db.connect()
 ```
+
 - Query execution: Accepts two parameters. The first is the query and the second is the list of parameters to be used. See example below
 
 ```python
-data = con.query(query, params)
+data = db.query(query, params)
 ```
+
 - Closing connection
 
 ```python
-con.close()
+db.close()
 ```
 
 **Sample config object**
+
 ```python
 conf = {
     'host': 'localhost',
     'user': 'root',
     'password':'',
     'db': 'mydb',
-    'port': 3306    
+    'port': 3306,
+    'max_pool_size': 20     # optional, default is 10
 }
 ```
 
 ##### SELECT operations
+
 ```python
 from nmi_mysql import nmi_mysql
 
-connection = nmi_mysql.DB(conf, True)
+db = nmi_mysql.DB(conf)
+db.connect()
 
-data1 = connection.query('SELECT * FROM mytable WHERE name = %s', ['ninz'])
-data2 = connection.query('SELECT * FROM mytable WHERE name IN (%s) AND age = %s', [['john', 'doe'], 10])
-
-connection.close()
+data1 = db.query('SELECT * FROM mytable WHERE name = %s', ['ninz'])
+data2 = db.query('SELECT * FROM mytable WHERE name IN (%s) AND age = %s', [['john', 'doe'], 10])
```

Comment:
For me, it's not good to the eyes. :) Also, it can be difficult to be tracked/debugged since multiple statements in one query will only return the first query results I think.
If we will not be concerned with the user in using this feature, then we can show some examples just to inform them that it's possible.